tests/testthat/test-check-all.R

```diff
@@ -0,0 +1,186 @@
+context("test-check-all.R")
+
+library("tibble")
+syn <- attempt_instantiate()
+attempt_login(syn)
+annots <- get_synapse_annotations(syn = syn)
```

Comment:
Can we use pre-constructed annotations to speed this up a little and make it simpler, something like this: 
https://github.com/Sage-Bionetworks/dccvalidator/blob/22fde0371c8daa9c2d1da8e6b80d881c1ee2a347/tests/testthat/test-check-annotation-keys.R#L7-L14